Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
Just a heads-up, you don't need to worry about uniqueness with Rails `to_param` as it does include the ID in the param. It just works. Obviously not as cool as FriendlyID, but a lot simpler.

Anyway, just a suggestion - I always prefer to use less gems if possible -- other opinions may differ :smile:. 

Another consideration is FriendlyID quickly pollutes the code base with:

` Event.friendly.find(param)`

But with a code base this size it probably doesn't matter.